core/wire/kem.go

```diff
@@ -27,19 +27,32 @@ import (
 	"os"
 	"strings"
 
+	"github.com/cloudflare/circl/kem/kyber/kyber768"
 	"github.com/katzenpost/nyquist/kem"
 	"github.com/katzenpost/nyquist/seec"
 	"golang.org/x/crypto/blake2b"
 
+	"github.com/katzenpost/katzenpost/core/crypto/kem/adapter"
+	"github.com/katzenpost/katzenpost/core/crypto/kem/hybrid"
+	"github.com/katzenpost/katzenpost/core/crypto/nike/ecdh"
 	cpem "github.com/katzenpost/katzenpost/core/crypto/pem"
+	"github.com/katzenpost/katzenpost/core/crypto/rand"
 )
 
 // PublicKeyHashSize indicates the hash size returned
 // from the PublicKey's Sum256 method.
 const PublicKeyHashSize = 32
 
+var KEMScheme = hybrid.New(
```

Comment:
Is this `var KEMScheme` exported here - is it used elsewhere? Looks like `DefaultScheme` ought to be enough?

Reply:
I want to export it so that we can thoroughly test it from inside of the core/crypto/kem module:
crypto/kem/schemes/kem_test.go

Reply:
but actually we don't need to define the kem scheme inside core/wire... probably better to define it in core/crypto/kem/hybrid

Reply:
hmm or maybe we do want the semantics of "run these kem tests on whatever kem the core/wire protocol uses"

Reply:
okay i've "fixed" it.